recipes/paml/4.9/baseml.ctl

```diff
@@ -0,0 +1,34 @@
+      seqfile = brown.nuc 
+     treefile = brown.trees
+
+      outfile = mlb       * main result file
+        noisy = 2   * 0,1,2,3: how much rubbish on the screen
+      verbose = 0   * 1: detailed output, 0: concise output

```

Comment:
@bgruening yes, but this program is a bit tricky in that it requires a valid input file from the command line, and does not support any easy parameters like --help, --version, etc.  If a valid file is not received, the program enters into a prompt that waits for user input - see here:
```
$ ./baseml --version
BASEML in paml version 4.9, March 2015

error when opening file --version
tell me the full path-name of the file?
```
This prompt hangs the travis test since it is waiting for user interaction.  The test commands with the small valid input datasets I've used are the only way I could find that allows the tests to complete without any interaction from the user.  Are there other packages that have this issue?  If so, is there a better way for me to handle this?